explorer/interpreter/interpreter.cpp

```diff
@@ -2396,11 +2408,8 @@ auto Interpreter::StepStmt() -> ErrorOr<Success> {
             value, heap_.Read(location->address(), ret_var.source_loc()));
       }
       const CallableDeclaration& function = cast<Return>(stmt).function();
-      CARBON_ASSIGN_OR_RETURN(
-          Nonnull<const Value*> return_value,
-          Convert(value, &function.return_term().static_type(),
-                  stmt.source_loc()));
-      return todo_.UnwindPast(*function.body(), return_value);
+      // TODO: Not needed?
```

Comment:
If I recall correct what unwind is doing, I expect this is still needed to deal with returning from a nested scope.